tst/EnergyPlus/unit/CurveManager.unit.cc

```diff
@@ -120,6 +120,52 @@ TEST_F(EnergyPlusFixture, QuadraticCurve)
 
     EXPECT_EQ(0., state->dataCurveManager->PerfCurve(1).CurveMin);
     EXPECT_TRUE(state->dataCurveManager->PerfCurve(1).CurveMinPresent);
+    
+    double var1 = 1, var2 = 0.1, var3 = 20, var4 = 10;
+    double expected_value = -3.3333 + (0.1*1) + (38.9*0.1) + (0.1*20) + (0.5 * 10);
+    EXPECT_EQ(expected_value, CurveManager::CurveValue(*state, 1, var1, var2, var3, var4));
```

Comment:
add curve value check for quad-linear curve bugfix (where 4th variable is ignored in calculations)